pkg/services/dashboardversion/dashverimpl/sqlx_store.go

```diff
@@ -59,16 +59,15 @@ func (ss *sqlxStore) DeleteBatch(ctx context.Context, cmd *dashver.DeleteExpired
 	return deleted, err
 }
 
-func (ss *sqlxStore) List(ctx context.Context, query *dashver.ListDashboardVersionsQuery) ([]*dashver.DashboardVersionDTO, error) {
-	var dashboardVersion []*dashver.DashboardVersionDTO
+func (ss *sqlxStore) List(ctx context.Context, query *dashver.ListDashboardVersionsQuery) ([]*dashver.DashboardVersion, error) {
+	var dashboardVersion []*dashver.DashboardVersion
 	qr := `SELECT dashboard_version.id,
 				dashboard_version.dashboard_id,
 				dashboard_version.parent_version,
 				dashboard_version.restored_from,
 				dashboard_version.version,
 				dashboard_version.created,
-				dashboard_version.message,
-				"user".login as created_by_login
```

Comment:
Sort of, and I'm certainly willing to re-think this. We're losing the `created_by_login` string from the sqlx store, which we weren't getting from the equivalent xorm query (and so I wasn't worried that any callers were depending on it - I couldn't find any references to the field); both are returning the `created_by` ID int which can we expand (with access to the user store) to include the login string. 

I could revert this, though I'd prefer to leave comments suggesting we change the DTO to include the login string directly once it's part of all of our store implementations. What do you think?

Reply:
🤔 This does look a bit wonky and it looks like the tests don't check those fields. I'll extend the store tests and see what's really going on there.

Reply:
I have _no_ idea about where that'd be used or if it's needed 🙈

Reply:
Ok, tests are looking good and the expected information is returned (I'll push those changes but I'm still adding tests at the moment); @ying-jeanne can you take a look at this change and let me know what you think? I replaced the `DashboardVersionDTO` object returned by `List` with the `DashboardVersion` object and since the xorm store implementation wasn't populating the CreatedBy (string) field (it's handled by [the api](https://github.com/grafana/grafana/blob/main/pkg/api/dashboard.go#L153-L160)) I left it out.

Reply:
I did double check after the discussion, indeed the left join on user table is just for login, if we cover it in the api, we can remove it here and also the left join 2 lines after.